<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>801</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>805</width>
      <height>91</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 4, 45);
background-color: rgb(213, 0, 4);
background-color: rgb(232, 0, 0);
background-color: rgb(191, 0, 0);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="nav">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>341</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>341</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color: white;
text-decoration: underline;
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="viewCases_button">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>VIEW CASES</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="comparativeReport_button">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>COMPARATIVE REPORT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pcr_button">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left: 12px;</string>
        </property>
        <property name="text">
         <string>PATIENT CARE REPORT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="nav2">
     <property name="geometry">
      <rect>
       <x>698</x>
       <y>9</y>
       <width>90</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="save_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
font: 11pt &quot;Impact&quot;;
text-decoration: underline;
color:rgb(17, 255, 0);</string>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="logout_button">
     <property name="geometry">
      <rect>
       <x>635</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
font: 11pt &quot;Impact&quot;;
text-decoration: underline;
color: rgb(255, 247, 0);</string>
     </property>
     <property name="text">
      <string>LOGOUT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_button">
     <property name="geometry">
      <rect>
       <x>564</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
font: 11pt &quot;Impact&quot;;
text-decoration: underline;
color: rgb(91, 0, 0);</string>
     </property>
     <property name="text">
      <string>DELETE</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="body">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>801</width>
      <height>511</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 0, 1);
background-color: rgb(179, 116, 116);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>18</number>
     </property>
     <property name="bottomMargin">
      <number>199</number>
     </property>
     <item>
      <widget class="QFrame" name="content1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(76, 0, 1);
color: rgb(255, 234, 73);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="date_label">
          <property name="text">
           <string>DATE:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="casenum_label">
          <property name="text">
           <string>CASE NO:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="casenum_input"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="location_label">
          <property name="text">
           <string>LOCATION:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="location_input"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="numkm_label">
          <property name="text">
           <string>NO. OF KM:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="numkm_input"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="incident_label">
          <property name="text">
           <string>TYPE OF INCIDENT:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="involed_label">
          <property name="text">
           <string>INVOLVED:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="involved_input"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="informant_label">
          <property name="text">
           <string>INFORMANT:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="informant_input"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="resteam_label">
          <property name="text">
           <string>RESPONDING TEAM:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="resteam_input">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="othagency_label">
          <property name="text">
           <string>OTHER AGENCY:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="othagency_input"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="dutyems_label">
          <property name="text">
           <string>DUTY EMS:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="dutyems_input"/>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="date_input">
          <property name="currentSection">
           <enum>QDateTimeEdit::DaySection</enum>
          </property>
          <property name="displayFormat">
           <string>dd/MMM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="incident_input">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(191, 0, 0);</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MEDICAL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MVA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TRAUMA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EMERGENCY</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QFrame" name="content2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(76, 0, 1);
color: rgb(255, 234, 73);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>40</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="calltime_label">
          <property name="text">
           <string>TIME OF CALL:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="stationout_label">
          <property name="text">
           <string>OUT OF STATION:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="scenearrive_label">
          <property name="text">
           <string>ARIVE AT SCENE:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="scenearrive_input">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="backstation_label">
          <property name="text">
           <string>BACK TO STATION:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="backstation_input">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="restime_label">
          <property name="text">
           <string>RESPONSE TIME:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="restime_input">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="calltime_input">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="stationout_input">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="content3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(76, 0, 1);
color: rgb(255, 234, 73);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>16</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="name_label">
          <property name="text">
           <string>NAME:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="name_input"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="age_label">
          <property name="text">
           <string>AGE:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="age_input"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="gender_label">
          <property name="text">
           <string>GENDER:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="devmiles_label">
          <property name="text">
           <string>DEV'T MILESTONE:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="devmiles_input"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="chcomplaints_label">
          <property name="text">
           <string>CHIEF COMPLAINTS:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="chcomplaints_input"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="genimpress_label">
          <property name="text">
           <string>GEN. IMPRESSION:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="genimpress_input"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="actionstake_label">
          <property name="text">
           <string>ACTIONS TAKEN:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="actionstake_input"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="hospref_label">
          <property name="text">
           <string>HOSPITAL REF.</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="hospref_input"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="gender_input">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(191, 0, 0);</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MALE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FEMALE</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>body</zorder>
   <zorder>header</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
